packages/ai/src/providers/google/index.ts

```diff
@@ -100,19 +114,10 @@ const createChatCompletion: ICreateChatCompletion = async (body) => {
       completionTokens = 0;
 
     try {
-      // Google's JS library does not fully support Gemini 1.5 Pro
-      // because of which the `countTokens` method needs to be requested
-      // via an older model. We have an open issue with details:
-      // https://github.com/google/generative-ai-js/issues/98
-      const tokenCounterModelInstance = model.includes("gemini-1.5")
-        ? googleAI.getGenerativeModel({
-            model: "gemini-pro",
-          })
-        : modelInstance;
       [{ totalTokens: completionTokens }, { totalTokens: promptTokens }] =
         await Promise.all([
-          tokenCounterModelInstance.countTokens(responseContent),
-          tokenCounterModelInstance.countTokens({
+          modelInstance.countTokens(responseContent),
```

Comment:
this will break with gemini-1.5-pro, right? need to use an older model to count tokens, even if the model=gemini-1.5-pro based on my testing

Reply:
The logic is same as before. I'm still getting the token counts without any errors

Reply:
Ran the CLI with gemini-1.0-pro and gemini-1.5-pro. Both are handled, added the screenshot in the PR description

Reply:
oh cool, thanks for checking that